coverage.xml

```diff
@@ -0,0 +1,176 @@
+<?xml version="1.0" ?>
```

Comment:
Is coverage a generated file from coverage or indeed part of the project?
If it's part of the project does it matter that it holds paths specific to your laptop?